[2024-03-04T15:04:55.058+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.get_data_task manual__2024-03-04T15:04:51.460208+00:00 [queued]>
[2024-03-04T15:04:55.064+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.get_data_task manual__2024-03-04T15:04:51.460208+00:00 [queued]>
[2024-03-04T15:04:55.064+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T15:04:55.065+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-03-04T15:04:55.065+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T15:04:55.072+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): get_data_task> on 2024-03-04 15:04:51.460208+00:00
[2024-03-04T15:04:55.076+0000] {standard_task_runner.py:55} INFO - Started process 175 to run task
[2024-03-04T15:04:55.078+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gera_dados_turbina', 'get_data_task', 'manual__2024-03-04T15:04:51.460208+00:00', '--job-id', '3332', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp9g0p1dix']
[2024-03-04T15:04:55.079+0000] {standard_task_runner.py:83} INFO - Job 3332: Subtask get_data_task
[2024-03-04T15:04:55.113+0000] {task_command.py:388} INFO - Running <TaskInstance: gera_dados_turbina.get_data_task manual__2024-03-04T15:04:51.460208+00:00 [running]> on host 61173039c7e2
[2024-03-04T15:04:55.151+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=adrianolima095@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gera_dados_turbina
AIRFLOW_CTX_TASK_ID=get_data_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-04T15:04:51.460208+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-04T15:04:51.460208+00:00
[2024-03-04T15:04:55.181+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-03-04T15:04:55.186+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gera_dados_turbina, task_id=get_data_task, execution_date=20240304T150451, start_date=20240304T150455, end_date=20240304T150455
[2024-03-04T15:04:55.210+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-04T15:04:55.228+0000] {taskinstance.py:2578} INFO - 3 downstream tasks scheduled from follow-on schedule check
